.gitignore

```diff
@@ -25,5 +25,6 @@ charts/redpanda/templates/external-service.yaml
 charts/redpanda/templates/external-tls-secret.yaml
 charts/redpanda/templates/some-users-updated.yaml
 charts/kminion/templates/redpanda-tls.yaml
+charts/connectors/templates/redpanda-tls.yaml
```

Comment:
Could we just move all these temporary files into a subdirectory and just exclude that directory?

Reply:
Agree. This PR will do it for connectors only. Rest will be done in different PR.